honeybadger/contrib/aws_lambda.py

```diff
@@ -82,18 +83,38 @@ def __init__(self):
     def supports(self, config, context):
         return config.is_aws_lambda_environment
 
-    def generate_payload(self, config, context):
+    def generate_payload(self, default_payload, config, context):
         """
         Generate payload by checking the lambda's
         request event
         """
-        payload = {
-            "params": {
-                "event": current_event()
-            },
+        request_payload = {
+            "event": current_event(),
             "context": {}
         }
-        return filter_dict(payload, config.params_filters)
+        default_payload["request"] = filter_dict(request_payload, config.params_filters)
```

Comment:
Is this going to override existing keys in `default_payload["request"]`, such as `default_payload["request"]["context"]`? We need to make sure all plugins preserve the existing context data that is passed in.

Reply:
Right... I have corrected that for the lambda plugin. It is preserved for others